source/inventory_api.rst

```diff
@@ -0,0 +1,461 @@
+************************************
+Inventory API
+************************************
+
+
+`1. Add / Update Inventory`_
+
+`2. Update Out of Stock Inventory`_
+
+
+1. Add / Update Inventory
+==================
+| URL endpoint: https://host.com/api/backend/inventories/update
+| Allowed method: POST
+| Requires authentication: Yes
+
+A. Request
+----------
+
+You can add or update a product variant in the store inventory by calling the API with these parameters
+
+Header
+______
+
+=================== =========== =======================
+Parameter           Required    Description
+=================== =========== =======================
+Content-Type        Yes         application/json
+Authorization       Yes         merchant token string
+=================== =========== =======================
+
+.. code-block::
+
+    Content-Type: application/json
+    Authorization: token fc9a483c04a6b66745b26741aa24f3d0d29b9d84
+
+Body
+______
+
+=================== =========== =======================
+Parameter           Required    Description
+=================== =========== =======================
+store_code          Yes         Store Code string
+inventories         Yes         An array of product variant objects
+=================== =========== =======================
+
+Each product variant object in the array has the following parameters 
```

Comment:
`Each variant`, `product` and `variant` is two different things

Reply:
I described it as a product variant since it is, a variant of a product, but I have changed it already.

Reply:
because 1 product can have multiple variants